javascript/packages/fury/lib/type.ts

```diff
@@ -28,7 +29,7 @@ export type BinaryReader = ReturnType<typeof BinaryReader>;
 export enum InternalSerializerType {
   STRING = 13,
   ARRAY = 25,
-  TUPLE = 25,
+  TUPLE = 25.1,
```

Comment:
after the enum value change, we can remove the redundant code now:
<img width="1581" alt="image" src="https://github.com/apache/incubator-fury/assets/13938334/54a7cfd9-3541-43bc-82b3-b33207cfab32">